pyproject.toml

```diff
@@ -158,13 +160,15 @@ disable = [
     "too-many-statements",
 ]
 
+
```

Comment:
Remove the empty line because the “pylint” contains multiple sections spaced with a single line. Same below.